docs/k0s-in-docker.md

```diff
@@ -1,55 +1,65 @@
-# Running k0s in Docker
+# Run k0s in Docker
 
-In this tutorial you'll create a k0s cluster on top of docker. By default, both controller and worker are run in the same container to provide an easy local testing "cluster". The tutorial also shows how to add additional worker nodes to the cluster.
+You can create a k0s cluster on top of docker. In such a scenario, by default,
+both controller and worker nodes are run in the same container to provide an
+easy local testing "cluster".
 
-### Prerequisites
+## Prerequisites
 
-Docker environment on Mac, Windows or Linux. [Get Docker](https://docs.docker.com/get-docker/).
+You will require a [Docker environment](https://docs.docker.com/get-docker/) running on a Mac, Windows, or Linux system.
 
-### Container images
+## Container images
 
-The k0s containers are published both on Docker Hub and GitHub. The examples in this page show Docker Hub, because it's more simple to use. Using GitHub requires a separate authentication (not covered here). Alternative links:
+The k0s containers are published both on Docker Hub and GitHub. For reasons of
+simplicity, the examples given here use Docker Hub (GitHub requires a separate
+authentication that is not covered). Alternative links include:
 
 - docker.io/k0sproject/k0s:latest
 - docker.pkg.github.com/k0sproject/k0s/k0s:"version"
 
-### Installation steps
+## Install k0s
```

Comment:
It's good that this title "Install k0s" is aligned within all tutorials. However, in this page "Install" is maybe a bit too much as it's just about starting k0s. There's nothing to actually install (valid for this page only).

Reply:
Changing to:

## Start k0s
### Initiate k0s